content/reference-ledger-format.md

```diff
@@ -398,3 +398,72 @@ The `index` of a RippleState node is the SHA-512Half of the following values put
+
+The `SignerList` node type represents a list of parties that, as a group, are authorized to sign a transaction in place of an individual account.
+
+Example SignerList node:
+
+```
+{
+    "Flags": 0,
+    "LedgerEntryType": "SignerList",
+    "OwnerNode": "0000000000000000",
+    "PreviousTxnID": "5904C0DC72C58A83AEFED2FFC5386356AA83FCA6A88C89D00646E51E687CDBE4",
+    "PreviousTxnLgrSeq": 16061435,
+    "SignerEntries": [
+        {
+            "SignerEntry": {
+                "Account": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+                "SignerWeight": 2
+            }
+        },
+        {
+            "SignerEntry": {
+                "Account": "raKEEVSGnKSD9Zyvxu4z6Pqpm4ABH8FS6n",
+                "SignerWeight": 1
+            }
+        },
+        {
+            "SignerEntry": {
+                "Account": "rUpy3eEg8rqjqfUoLeBnZkscbKbFsKXC3v",
+                "SignerWeight": 1
+            }
+        }
+    ],
+    "SignerListID": 0,
+    "SignerQuorum": 3,
+    "index": "A9C28A28B85CD533217F5C0A0C7767666B093FA58A0F2D80026FCC4CD932DDC7"
+}
+```
+
+A SignerList node has the following fields:
+
+| Name            | JSON Type | Internal Type | Description |
+|-----------------|-----------|---------------|-------------|
+| OwnerNode       | String    | UInt64        | A hint indicating which page of the owner directory links to this node, in case the directory consists of multiple nodes. |
+| SignerQuorum    | Number    | UInt32        | A target number for signer weights. To produce a valid signature for the owner of this SignerList, the signers must provide valid signatures whose weights sum to this value or more. |
+| SignerEntries   | Array     | Array         | An array of SignerEntry objects representing the parties who are part of this signer list. |
+| SignerListID    | Number    | UInt32        | An ID for this signer list. Currently always set to `0`. If a future update allows multiple signer lists for an account, this may change. |
+| PreviousTxnID   | String    | Hash256       | The identifying hash of the transaction that most recently modified this node. |
+| PreviousTxnLgrSeq | Number  | UInt32        | The sequence number (`ledger_index`) of the ledger that contains the transaction that most recently modified this node. |
+
+The `SignerEntries` may be any combination of funded and unfunded addresses that use either secp256k1 or ed25519 keys.
+
+### SignerEntry object ###
+
+Each member of the `SignerEntries` field is an object that describes that signer in the list. A SignerEntry has the following fields:
+
+| Name            | JSON Type | Internal Type | Description |
+|-----------------|-----------|---------------|-------------|
+| Account         | String    | AccountID     | An address whose signature contributes to the multi-signature. This does not need to be a funded Ripple account. |
```

Comment:
Interesting. Since you said it "does not need to be a funded Ripple ~~account~~ address," it made me wonder if anything other than a *Ripple* address (or account or whatever) could sign. (I'm pretty sure the answer is no.) But, yeah, also probably a little clearer to use address than account, too.